### ISSUE TITLE
[BUG] Type error on editing Scheduler task for Re-Index

### ISSUE THREAD

**Issue by JBlocks** [TYPO3-Solr/ext-solr#3234]
**Describe the bug**
If you want to edit an existing scheduler task (Force Re-Index), a type error occurs.

**To Reproduce**
1. Go to 'Module Scheduler'
2. Click on 'existing task for re-index'
3. See argument type error: `ApacheSolrForTypo3\Solr\Domain\Site\SiteRepository::getSiteByRootPageId() must be of the type int, string given,`

**Expected behavior**
The type error doesn't occur and the pageId is of type int.

**Used versions:**
 - TYPO3 Version: 11.5.9
 - Browser: chrome
 - EXT:solr Version: dev-release-11.5.x
 - PHP Version: 7.4

**Additional context**
```
Argument 1 passed to ApacheSolrForTypo3\Solr\Domain\Site\SiteRepository::getSiteByRootPageId() must be of the type int, string given, called in /var/www/html/Source/web/typo3conf/ext/solr/Classes/Task/ReIndexTaskAdditionalFieldProvider.php on line 109
```

**Comment by mediaessenz** [comment]
Can easily fixed by casting value in line 109 of Classes/Task/ReIndexTaskAdditionalFieldProvider.php:
```php
$this->site = $this->siteRepository->getSiteByRootPageId((int)$task->getRootPageId());
```

**Comment by dkd-kaehm** [comment]
Fixed in #3238 https://github.com/TYPO3-Solr/ext-solr/pull/3238/commits/f467e5ae9a8f45b892cf73a321a671fffd7e9d88
